### PR TITLE
Finishing mobile version web

### PR DESCRIPTION
**Pull request by yaredtekle022**
### These lists of activities were done

- Added works section
- Added About me section
- set linters

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,406 @@
+@media only screen and (max-width: 765px) {
+  body {
+    margin: 0;
+    background-color: black;
```
Comment (Nyame-Wolf) [inline styles.css]: Kindly note that it's not the whole document that has a black background. I suggest you add the background color to only the sections that need it, that is the `headline` and `about`

This are examples of parts of the design that have a different background:
Screenshot 2022-12-02 at 14 32 33

File: index.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
```
Comment (Nyame-Wolf) [inline index.html:129]: I suggest adding background images just as the figma design has. For this project, part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) was to stick as much as possible to the design
Screenshot 2022-12-02 at 14 39 40

File: index.html [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
```
Comment (Nyame-Wolf) [inline index.html:151]: I suggest adding background images just as the figma design has. For this project, part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) was to stick as much as possible to the design
Screenshot 2022-12-02 at 14 39 40

File: index.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
```
Comment (Nyame-Wolf) [inline index.html:173]: I suggest adding background images just as the figma design has. For this project, part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) was to stick as much as possible to the design
Screenshot 2022-12-02 at 14 39 40

File: index.html [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+      <div class="profesion">
```
Comment (Nyame-Wolf) [inline index.html:218]: I suggest adding background images just as the figma design has. For this project, part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) was to stick as much as possible to the design
Screenshot 2022-12-02 at 14 39 40

File: index.html [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
```
Comment (Nyame-Wolf) [inline index.html:193]: I suggest for these, make sure they match the figma design. This is how your project currently looks:
Screenshot 2022-12-02 at 14 43 13
this is how the figma design looks
Screenshot 2022-12-02 at 14 32 44

File: index.html [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+        <button class="project-button" href="#">See Project</button>
+      </div>
+    </section>
+    <footer>
+      <section class="about">
```
Comment (Nyame-Wolf) [inline index.html:242]: Kindly add the background image that is in the figma design in the about section.
This is your design
Screenshot 2022-12-02 at 14 43 41
This is the figma design:
Screenshot 2022-12-02 at 14 47 28

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @yaredtekle022  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by OyePriscilla — APPROVED** [review]
Hi @yaredtt 👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Good commit messages ✔️ 
GitHub flow observed 😄 

## Optional suggestions
[OPTIONAL] Please for subsequent PRs, kindly ensure that your commit messages are in sentence case. Thanks 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**